libs/etterlatte-funksjonsbrytere/src/main/kotlin/no/nav/etterlatte/funksjonsbrytere/FeatureToggleService.kt

```diff
@@ -0,0 +1,74 @@
+package no.nav.etterlatte.funksjonsbrytere
+
+import io.getunleash.DefaultUnleash
+import io.getunleash.UnleashContext
+import io.getunleash.UnleashContextProvider
+import io.getunleash.strategy.GradualRolloutRandomStrategy
+import io.getunleash.util.UnleashConfig
+import org.slf4j.LoggerFactory
+import java.net.URI
+
+fun initialiser(env: Map<String, String>): FeatureToggleService {
+    val enabled = env.getOrDefault(Properties.ENABLED.navn, false) as Boolean
+    return if (enabled) {
+        UnleashFeatureToggleService(
+            Unleash(
+                enabled = true,
+                uri = URI(env[Properties.URI.navn] ?: throw IllegalArgumentException("Unleash-URI er ikke definert")),
+                cluster = env[Properties.CLUSTER.navn]
+                    ?: throw IllegalArgumentException("Unleash-cluster er ikke definert"),
+                applicationName = env[Properties.APPLICATIONNAME.navn]
+                    ?: throw IllegalArgumentException("Unleash-applikasjonsnavn er ikke definert")
+            )
+        )
+    } else {
+        DummyFeatureToggleService()
+    }
+}
+
+interface FeatureToggleService {
+    fun isEnabled(toggleId: FeatureToggle, defaultValue: Boolean): Boolean
```

Comment:
A thought - should we have similar behaviour to unleash that if you don't override default then its false?

```
fun isEnabled(toggleId: FeatureToggle, defaultValue: Boolean = false): Boolean
```

Reply:
Makes sense to me

Reply:
På den andre sida er erfaringa frå andre stadar at vi lagar ein del toggles som bør vera false som standard og ein del toggles som bør vera true, så det har ein verdi å tvinge utviklar til å vera eksplisitt på det.

Reply:
Holder med det siste forslaget til Mads her.